include/glow/Importer/Caffe2ModelLoader.h

```diff
@@ -107,6 +110,14 @@ class Caffe2ModelLoader
 
   friend class ONNXIFIModelLoader;
 
+  /// \returns success if the folding of operator \p op in the loader
+  /// \p pLoader is successful. The folding utility uses temporary
+  /// loader \p loaderCF, and associated temporary function \p F.
+  template <class LoaderType, class OpType>
+  friend llvm::Error constantFoldInLoader(Function *F, LoaderType &loaderCF,
```

Comment:
Can you also use just `tmpLoader` and `loader` instead of `loaderCF` and `pLoader`?